src/SwedbankPay/Core/Core.php

```diff
@@ -13,13 +15,14 @@
 use SwedbankPay\Core\Library\Methods\Checkout;
 use SwedbankPay\Core\Library\Methods\Consumer;
 
-class Core implements CoreInterface
+class Core implements CoreInterface, MobilepayInterface
```

Comment:
Why does `Core` implement the `MobilePayInterface` but none of the other payment instrument interfaces?

Reply:
@asbjornu Because that isn't ready yet, that in TODO list :-)